Update the description of kopAllowedNamespaces (#1468)
streamnative/kop: docs/configuration.md

| kafkaMetadataTenant    | The tenant used for storing Kafka metadata topics    | public  |
| kafkaEnableMultiTenantMetadata    | Use the SASL username as `kafkaMetadataTenant`  | true  |
| kafkaMetadataNamespace | The namespace used for storing Kafka metadata topics | __kafka |

> | kopAllowedNamespaces   | The allowed namespace to list topics with a comma separator.<br>For example, `kopAllowedNamespaces=public/default,public/kafka`.<br>If not configured or empty, the allowed namespaces will take values from `kafkaTenant` and `kafkaNamespace`, which will be, `<kafkaTenant>/<kafkaNamespace>`. | |

Reviewer: ```suggestion
| kopAllowedNamespaces   | The allowed namespace to list topics with a comma separator.<br>For example, `kopAllowedNamespaces=public/default,public/kafka`.<br>If it is not configured or is empty, the allowed namespaces will get values from `kafkaTenant` and `kafkaNamespace`, which is `<kafkaTenant>/<kafkaNamespace>`. | |
```
